bundles/org.openhab.ui/web/src/css/app.styl

```diff
@@ -140,6 +140,15 @@ html
    .delete
      --f7-bars-link-color #ff3b30
 
+// Fix actions-modal/popup is placed behind the notch or outside of the screen on iPhone in openHAB iOS app
+@media (orientation: portrait)
+  .ios .actions-modal
```

Comment:
The class `.ios` denotes the ios _theme_, you probably want this applied for iOS _devices_ regardless of the theme. So that would be `.device-ios` instead.